src/index.html

```diff
@@ -1,17 +1,192 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+    onsubmit="onSubmit()"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname" class="field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </label>
+
+        <label for="name-field" class="field">
+          Name:
+          <input
+            type="text"
+            name="name-field"
+            id="name-field"
+          >
+        </label>
+
+        <label for="age" class="field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="0"
+            max="0"
```

Comment:
```suggestion
            min="0"
            max="100"
```
Why do you use `min` and `max` 0? There should be some range for age srelection